checkers/python/hashids-with-flask-secret.yml

```diff
@@ -0,0 +1,28 @@
+language: py
+name: hashids-with-flask-secret
+message: Do not use Flask secret key as salt in HashIDs
```

Comment:
Python's hashids package has been renamed to [`sqids`](https://sqids.org/python). We should handle this for both imports.

On another note, maybe we should make this checker generic and handle other libraries as well that people use for creating hashed identifiers?